src/ethereum/info/avaxInfo.ts

```diff
@@ -1,8 +1,220 @@
-import { EdgeCurrencyInfo } from 'edge-core-js/types'
+import { EdgeCurrencyInfo, EdgeTokenMap } from 'edge-core-js/types'
 
 import { makeOuterPlugin } from '../../common/innerPlugin'
+import { getEvmTokenId, makeTokenMap } from '../../common/tokenHelpers'
 import type { EthereumTools } from '../ethPlugin'
-import { EthereumFees, EthereumSettings } from '../ethTypes'
+import type { EthereumFees, EthereumSettings } from '../ethTypes'
+
+const builtinTokens: EdgeTokenMap = makeTokenMap(getEvmTokenId, [
```

Comment:
Hmm. I'm not liking that the "source of truth" is in a weird array format that's neither the old or new EdgeTokenMap. Why not just use an EdgeTokenMap? That's how XRP is coded.

Reply:
Admittedly you'd want some code to automate the conversion the first time but then you're done and can commit the results. Given that the format of the EdgeMetaToken array and the new array are different, I presume you did use automation to do the conversion. Might as well go straight to a token map

Reply:
I did the conversion using regular expression search & replace. For the `EdgeTokenMap` conversion, I had to do it by hand. Now that it's done, we have unit tests to ensure that it always stays correct going forward.